src/views/helpers/jobPrefIsPublic.js

```diff
@@ -0,0 +1,9 @@
+module.exports = (jobViewPref) => {
+  switch (jobViewPref) {
+    case 'public':
+      return true;
+      break;
+    default:
+      return false;
+  }
+}
```

Comment:
I think if we only have 2 options we shouldn't use a switch case as if/else syntax will be clearer.